recipes/breakpad/all/test_package/test_package.cpp

```diff
@@ -17,8 +18,7 @@ bool callback(const MinidumpDescriptor &descriptor,
 }
 
 int main(int argc, char *argv[]) {
-
-    std::cout << "Breakpad test_package" << std::endl;
+    std::cout << "Breakpad test_package\n";
```

Comment:
Why this?

Reply:
No particular reason other then no no need for the flush.